EarlGrey/Additions/UIViewController+GREYAdditions.m

```diff
@@ -138,19 +138,14 @@ - (void)greyswizzled_viewWillAppear:(BOOL)animated {
       // Interactive transitions can cancel and cause imbalance of will and did calls.
       id<UIViewControllerTransitionCoordinator> coordinator = [self transitionCoordinator];
       if (coordinator && [coordinator initiallyInteractive]) {
-        void (^contextBlock)(id<UIViewControllerTransitionCoordinatorContext>) =
-            ^(id<UIViewControllerTransitionCoordinatorContext> context) {
-              if ([context isCancelled]) {
-                id object =
-                    objc_getAssociatedObject(self, @selector(greyswizzled_viewWillAppear:));
-                UNTRACK_STATE_FOR_OBJECT(kGREYPendingViewsToAppear, object);
-              }
-            };
-#if !defined(__IPHONE_10_0) || (__IPHONE_OS_VERSION_MIN_REQUIRED < __IPHONE_10_0)
```

Comment:
I think you should keep these

Reply:
This is deleted in google3 repo. Any special reason we need to keep it?

Reply:
Yes, because we never synced it into google after it was committed externally here:
https://github.com/google/EarlGrey/commit/2fd8a7d4b76f820fb95bce495c0ceb324dbe3edb